functions/opencast-op-counts.py

```diff
@@ -0,0 +1,50 @@
+import mysql.connector
+from os import getenv as env
+import urllib.parse as urlparse
+
+import logging
+from common import setup_logging
+logger = logging.getLogger()
+
+OPENCAST_DB_URL = env('OPENCAST_DB_URL');
```

Comment:
Super low importance but I'm trying to use double quotes consistently instead of single quotes.

Reply:
:+1: 

I started to do `tasks.py` too but it was too much. Would blow up the diff and potentially introduce bugs in stuff that we wouldn't find until too late.